c++/src/kj/exception.c++

```diff
@@ -632,6 +636,7 @@ void printStackTraceOnCrash() {
   stack.ss_sp = reinterpret_cast<char*>(mmap(
       nullptr, stack.ss_size, PROT_READ | PROT_WRITE,
       MAP_ANONYMOUS | MAP_PRIVATE | MAP_GROWSDOWN, -1, 0));
+#if !defined(__EMSCRIPTEN__)
```

Comment:
In KJ code we usually don't use `defined()`, but instead rely on the fact that undefined identifiers evaluate to zero for the purpose of preprocessor directives. So I'd prefer `#if !__EMSCRIPTEN__` rather than `#if !defined(__EMSCRIPTEN__)`.